tutorials/xarray-spatial_classification-methods.ipynb

```diff
@@ -0,0 +1,372 @@
+    "img_arr = xr.open_rasterio(url).squeeze().drop(\"band\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b6ae27af",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "img_arr.plot.imshow(figsize=(15, 10));"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "42c128aa",
+   "metadata": {},
+   "source": [
+    "### 2. Classify elevation data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d1f4e54",
+   "metadata": {},
+   "source": [
+    "#### Classify with `natural_breaks()`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "41759319",
+   "metadata": {},
+   "source": [
+    "Use the [natural breaks](https://xarray-spatial.org/reference/_autosummary/xrspatial.classify.natural_breaks.html) function to classify data with the [Jenks natural breaks classification](http://wiki.gis.com/wiki/index.php/Jenks_Natural_Breaks_Classification) method. This method is designed to distribute data into classes according to clusters that form a \"natural\" group within the data. The algorithm minimizes the average deviation from the class mean while also maximizing the deviation from the means of the other groups. Therefore, it is generally not recommended for data with low variance."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1c5d7b05",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datashader.transfer_functions import shade\n",
+    "import matplotlib.pyplot as plt\n",
+    "from xrspatial.classify import natural_breaks\n",
+    "\n",
+    "natural_breaks_agg = natural_breaks(img_arr, num_sample=20000, k=15)\n",
+    "\n",
+    "shade(natural_breaks_agg, cmap=plt.get_cmap(\"terrain\"), how=\"linear\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "40f7ee45",
+   "metadata": {},
+   "source": [
+    "You can see in the image above that the different elevation levels of the Grand Canyon are now identified by a limited number of distinct colors. Each color represents a range of values within the classified data. For example, the dark blue color areas represent the smallest elevation levels with around 700m of altitude, the yellow color areas represent elevation levels of around 1700m, and the white color areas represent the highest elevations of around 2500m of altitude."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4797d619",
+   "metadata": {},
+   "source": [
+    "#### Classify with `equal_interval()`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "baed3e1b",
+   "metadata": {},
+   "source": [
+    "To classify data into sets based on intervals of equal width, use the [equal interval](https://xarray-spatial.org/reference/_autosummary/xrspatial.classify.equal_interval.html) function. The [equal interval classification](http://wiki.gis.com/wiki/index.php/Equal_Interval_classification) is useful in cases where you want to emphasize the amount of an attribute value relative to the other values."
```

Comment:
I'm not sure I understand what "amount of an attribute value" means. Maybe rephrase it / expand on it?

Reply:
@TomAugspurger hey yes

- [ ] rephrase "amount of an attribute value"
- [ ] explain natural breaks agg
- [ ] merge adjacent cells
- [ ] guidance on choosing classificaion parameters